File: Master details_blazor-wasm/Master details/Models/Employee.cs
namespace Master_details.Models
{
    public class EmployeeModel
    {
        public int EmployeeID { get; set; }

        public string AvatarUrl { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Title { get; set; }

        public Address Address { get; set; }
    }
}

File: angular/src/app/models/employee.ts
export interface Employee {
    "employeeID": number,
    "lastName": string,
    "firstName": string,
    "title": string,
    "titleOfCourtesy": string,
    "birthDate": Date,
    "hireDate": Date,
    "address": string,
    "city": string,
    "region": string | null,
    "postalCode": string,
    "country": string,
    "homePhone": string,
    "extension": string,
    "notes": string,
    "reportsTo": number,
    "photoPath": string
  }
  